src/fides/lib/models/fides_user_permissions.py

```diff
@@ -1,34 +1,17 @@
-from typing import List, Tuple
-
 from sqlalchemy import ARRAY, Column, ForeignKey, String
 from sqlalchemy.orm import backref, relationship
 
 from fides.lib.db.base_class import Base
 from fides.lib.models.fides_user import FidesUser
-from fides.lib.oauth.privileges import privileges
-from fides.api.ops.api.v1.scope_registry import PRIVACY_REQUEST_READ
 
 
 class FidesUserPermissions(Base):
     """The DB ORM model for FidesUserPermissions"""
 
     user_id = Column(String, ForeignKey(FidesUser.id), nullable=False, unique=True)
-    # escaping curly braces requires doubling them. Not a "\". So {{{test123}}}
-    # renders as {test123}
-    scopes = Column(
-        ARRAY(String), nullable=False, default=f"{{{PRIVACY_REQUEST_READ}}}"
-    )
+    scopes = Column(ARRAY(String), nullable=True, server_default="{}", default=dict)
+    roles = Column(ARRAY(String), nullable=True, server_default="{}", default=dict)
```

Comment:
I'm making scopes nullable here - the user can have scopes or roles.   I've also updated the default value - the formatting of `{{{PRIVACY_REQUEST_READ}}}` doesn't work as intended, but wasn't obvious because we were always supplying scopes when creating permissions - it was never left as a default.  But this default could create scopes as ["{", "P", "R", "I", "V", "A", "C", "Y"...]